installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc

```diff
@@ -38,7 +38,6 @@ The Agent-based Installer can also optionally generate or accept Zero Touch Prov
 |`s390x`
 |&#10003;
 |&#10003;
-|ISO boot is not supported. Instead, use PXE assets.
```

Comment:
I would remove the entire column as this was the only comment. 

@skopacz1 Do you agree?

Reply:
Yep that's actually a very good point, I didn't look at this table closely enough. Thanks. 

In that case, the following parts should also be deleted:

- `|Comments` in line 21 (but just that part and not the entire line)
- line 26
- line 36